src/sentry/static/sentry/app/components/group/suggestedOwners/suggestedAssignees.tsx

```diff
@@ -24,7 +25,7 @@ type Props = {
 const SuggestedAssignees = ({owners, onAssign}: Props) => (
   <Wrapper>
     <Header>
-      <Heading>{t('Suggested Assignees')}</Heading>
+      <Heading is="h6">{t('Suggested Assignees')}</Heading>
```

Comment:
Do you think we could add 'named' heading styles to the heading module in the future? For example our sidebars generally use the same heading, and it would be nice to be able to do 
```jsx
import {SidebarHeader} from 'app/components/headings';
```

Reply:
I am thinking about having variants, such like variant="sidebar". I am going to talk to @doralchan about it 😉

Reply:
Having separate imports would be my preference as we wouldn't have to expose a more complex set of props that could interact poorly. Named components also makes usage simpler as the jsx is terser.